utils/users.py

```diff
@@ -93,6 +97,12 @@ async def verify_user(message):
 
         print(f"Verified user {user} with email {target_email}.")
         await logChan.send(embed=interact.send_embed('custom',"Verified User",f"{message.author} verified for megapod(s) {userInfo['megapods']} and pod(s) {userInfo['pods']}."))
+    # ValueError for missing user information
+    except ValueError as ve:
+        print(f"Verification failed for {message.author} with email {message.content}: {ve}")
+        await logChan.send(embed=interact.send_embed('custom', "Failed Verification", f"{message.author} tried to verify with email {message.content}. Reason: {ve}"))
+        await message.channel.send(f"Verification failed: {ve}")
```

Comment:
We don't want to send a message in the verify channel. The logchan is enough. you can remove line 104